src/Cdn/Cdn/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 5.1.0
```

Comment:
Please remove this line of version, make sure new entries go directly below line 21. This is because the next version will be calculated automatically during release process.